classes/ETL/Ingestor/HostTableTransformIngestor.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace ETL\Ingestor;
+
+use ETL\aOptions;
+use ETL\iAction;
+use ETL\aAction;
+use ETL\Configuration\EtlConfiguration;
+use ETL\EtlOverseerOptions;
+
+use Log;
+
+class HostTableTransformIngestor extends pdoIngestor implements iAction
+{
+
+    private $_instance_state;
+    private $_end_time;
```

Comment:
Please remove this $_end_time variable and the code that references it - it is not needed in this code.